docs/releasing.md

```diff
@@ -24,6 +24,13 @@ Creating a federation v2 release involves the following steps:
    3. `kubefed2 version` (check that the output is as expected)
    4. `tar cvzf kubefed2.tar.gz kubefed2`
    5. `sha256sum kubefed2.tar.gz > kubefed2.tar.gz.sha`
+4. Package the helm chart for release
+   1.  Adjust the default image tag in values.yaml
+   2.  Update the chart version in Chart.yaml
+   3.  Package the chart using `helm package federation-v2`
+   4.  Place the release packaged chart (.tgz file) in the federation-v2/ folder.
```

Comment:
This line should be replaced with

`sha256sum federation-v2-chart.tar.gz > federation-v2-chart.tar.gz.sha`

Reply:
Updated `   4.  `sha256sum federation-v2-<x.x.x>.tgz > federation-v2-<x.x.x>.tgz` (Name and checksum the packaged chart accordingly)`